src/harness/testcases/WINNF_FT_S_REG_testcase.py

```diff
@@ -30,6 +30,38 @@ def setUp(self):
   def tearDown(self):
     pass
 
+  def _convertRequestToRequestWithCpiSignature(self, private_key, cpi_id,
+                                               cpi_name, request,
+                                               jwt_algorithm='RS256'):
+    """Converts a regular registration request to contain cpiSignatureData
+       using the given JWT signature algorithm.
+
+    Args:
+      private_key: (string) valid PEM encoded string.
+      cpi_id: (string) valid cpiId.
+      cpi_name: (string) valid cpiName.
+      request: individual CBSD registration request (which is a dictionary).
+      jwt_algorithm: (string) algorithm to sign the JWT, defaults to 'RS256'.
+    """
+    cpi_signed_data = {}
+    cpi_signed_data['fccId'] = request['fccId']
+    cpi_signed_data['cbsdSerialNumber'] = request['cbsdSerialNumber']
+    cpi_signed_data['installationParam'] = request['installationParam']
+    del request['installationParam']
+    cpi_signed_data['professionalInstallerData'] = {}
+    cpi_signed_data['professionalInstallerData']['cpiId'] = cpi_id
+    cpi_signed_data['professionalInstallerData']['cpiName'] = cpi_name
+    cpi_signed_data['professionalInstallerData'][
+        'installCertificationTime'] = datetime.utcnow().strftime(
+            '%Y-%m-%dT%H:%M:%SZ')
+    compact_jwt_message = jwt.encode(
+        cpi_signed_data, private_key, jwt_algorithm)
+    jwt_message = compact_jwt_message.split('.')
+    request['cpiSignatureData'] = {}
+    request['cpiSignatureData']['protectedHeader'] = jwt_message[0]
+    request['cpiSignatureData']['encodedCpiSignedData'] = jwt_message[1]
+    request['cpiSignatureData']['digitalSignature'] = jwt_message[2]
+
```

Comment:
This PR must be old because this has moved to util.py already.  Please remove to make consistent but everything else looks fine!